packages/tldraw/src/lib/shapes/image/ImageShapeUtil.tsx

```diff
@@ -171,11 +180,15 @@ export class ImageShapeUtil extends BaseBoxShapeUtil<TLImageShape> {
 	}
 
 	override async toSvg(shape: TLImageShape) {
-		const asset = shape.props.assetId ? this.editor.getAsset(shape.props.assetId) : null
-
+		const zoomLevel = useValue('zoom level', () => this.editor.getZoomLevel(), [this.editor])
```

Comment:
imo svg exports shouldn't depend on the editor's zoom level - we usually just assume zoom = 100% for exports